### PR TITLE
Created flask_notworking and moved files there that doesn't work 

### PR DESCRIPTION
**Pull request by lsetiawan**
I moved the stuff that are not being used in `examples/flask` to `examples/flask_notworking`. Let me know if I miss anything. `cbi` and `swis` are currently used in travis ci and `odm1_1` and `odm2` are the ones we are focusing. 

This PR is related to ISSUE [#72](https://github.com/ODM2/WOFpy/issues/72#issuecomment-296416841) 

> split off into a new directory ("wof/examples/flask_notworking") the DAOs (directories) that have not been tested recently and will not be tested soon to ensure they work. For now, we're preserving and focusing only odm2 and odm_1_1)

### CODE REVIEW

File: wof/examples/flask_notworking/barebones/LCM_dao.py [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 
-import datetime
-
 import configparser
-from sqlalchemy import create_engine, distinct, func
-from sqlalchemy.orm import mapper, scoped_session, sessionmaker
-from sqlalchemy.sql import and_
 from dateutil.parser import parse
 from dateutil.tz import tzutc, tzoffset
+from sqlalchemy import create_engine, func
```
Comment (lsetiawan) [inline wof/examples/flask_notworking/barebones/LCM_dao.py]: Not sure why `distinct` got removed here. All I did was moved the whole folder.

File: wof/examples/flask_notworking/barebones/LCM_dao.py [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
+from sqlalchemy.orm import scoped_session, sessionmaker
```
Comment (lsetiawan) [inline wof/examples/flask_notworking/barebones/LCM_dao.py]: `mapper` also got removed here... weird...

**Comment by emiliom** [comment]
@lsetiawan, there was a discussion mostly with @ocefpaf, probably on a PR, where we decided to not do that "not working" directory separation after all. I don't remember which PR it was, so I can't point to the discussion.

I think I'm still on the balance between the two approaches, but I'll go with what we have now rather than accepting your PR. But first let me know if you have new arguments to make us go in this direction.

Thanks.

**Comment by lsetiawan** [comment]
@emiliom I guess I missed that memo. I don't have strong preference here. I'll close this PR then. Thanks.

**Comment by ocefpaf** [comment]
Just to clarify. Anyone using `master` should know that the code comes without any warranty. So keeping all the examples in the same location is the saner option for the devs.

However, we should not ship broken code, so I sent PR #103 to update the `MANIFEST.in` file and prevent those examples from making into the source distribution tarball.

**Comment by emiliom** [comment]
Thanks for the reminder and explanation, @ocefpaf!

**Pull request closed without merging**